scripts/qa_crowbarsetup.sh

```diff
@@ -1747,6 +1750,46 @@ EOF
     return 0
 }
 
+function install_crowbar_init()
+{
+    local apacheconfdir=/etc/apache2/conf.d
+
+    # FIXME: this is temporary until the package is doing that
+    # it is currently disabled by default not to break Crowbar
+    ln -sf $apacheconfdir/crowbar-sinatra.conf.partial $apacheconfdir/crowbar.conf.partial
+    systemctl start crowbar-init
+    wait_for 100 3 "onadmin_is_crowbar_init_api_available" "crowbar init service to start"
+}
+
+function onadmin_bootstrapcrowbar()
+{
+    # temporarily make it possible to not use postgres until we switched to the new upgrade process
+    # otherwise we would break the upgrade gating
+    [[ $want_postgres = 0 ]] && return
+    if iscloudver 7plus ; then
+        local http_code
+        install_crowbar_init
+        if $want_external_postgres ; then
+            # TODO: setup_external_postgres
+            # curl -X POST --data "username=crowbar&password=crowbar&host=$postgres_ip&port=$postgres_port&database=$postgres_dbname" $crowbar_init_api/database/connect
+            :
```

Comment:
If this is not implemented, I would sugest to remove from this PR and add in in a different PR (or in a different commit in this PR if is soon), also remove the var